Wangscape/AlphaCalculator.cpp

```diff
@@ -0,0 +1,85 @@
+#include "AlphaCalculator.h"
+#include <algorithm>
+#include <numeric>
+
+
+
+AlphaCalculator::AlphaCalculator()
+{
+}
+
+
+AlphaCalculator::~AlphaCalculator()
+{
+}
+
+void AlphaCalculator::calculatePixelAlphaLinear(WeightVector& weights, AlphaVector& alphas)
```

Comment:
I agree, this isn't the ideal way to store this data or transform it. In future there will be more stages of transformation between noise evaluation and tile generation (#37), and more textures to be blended. So the way data is passed through this function will probably change significantly.